src/fides/cli/commands/util.py

```diff
@@ -146,9 +146,18 @@ def up(
     """
 
     check_virtualenv()
-    check_docker_version()
+    try:
+        check_docker_version()
+    except:  # pylint: disable=bare-except
+        response = click.confirm(
```

Comment:
so this will actually now catch the exception thrown in the case where the version is found, but determined to be too low, right? i'm not against allowing the escape hatch in that scenario, feels like why not - but is this message going to be a bit confusing to make it seem like there was an error rather than just "correctly" determining the version was too low?

granted, i haven't tested this myself so i'm not exactly sure what the output/DX is, it may be perfectly easy to understand! i just wanted to pose the question

Reply:
I posted an example of the differences up above. This will preserve showing the proper Error message if there is a version mismatch, but if _any_ exception is thrown will still give the ability to forge ahead (YOLO)

Reply:
that looks good to me 👍 thank you for the screenshots/confirmation!